FirebaseRemoteConfig/Sources/RCNDevice.m

```diff
@@ -19,9 +19,9 @@
 #import <sys/utsname.h>
 
 #import <GoogleUtilities/GULAppEnvironmentUtil.h>
-#import "FirebaseCore/Sources/Public/FirebaseCore/FIRVersion.h"
 #import "FirebaseRemoteConfig/Sources/Private/RCNConfigSettings.h"
 #import "FirebaseRemoteConfig/Sources/RCNConfigConstants.h"
+#import "FIRVersion.h"
```

Comment:
I think it should be either
```
#import "FirebaseCore/Sources/Private/FirebaseCoreInternal.h"
```
or
```
#import <FirebaseCore/FirebaseCore.h>
```

Reply:
Yes, either should work, but the normal pattern in the repo is the former.

Reply:
I misread the docs! I caught the second half of the exception but not the first. I had the first suggestion you listed before but then thought I followed the right rule... whoops :) nice catch, thanks.